pkg/apis/servicecatalog/install/install.go

```diff
@@ -0,0 +1,30 @@
+// Package install registers the service-catalog API group
+package install
+
+import (
+	"k8s.io/kubernetes/pkg/apimachinery/announced"
+
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1"
+)
+
+func init() {
+	if err := announced.NewGroupMetaFactory(
+		&announced.GroupMetaFactoryArgs{
+			GroupName:              servicecatalog.GroupName,
+			VersionPreferenceOrder: []string{v1alpha1.SchemeGroupVersion.Version},
+			ImportPrefix:           "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog",
+			// TODO: what does this do?
```

Comment:
there's some reflect in this path that tries to pull out type names and needs to know which packages it came from to properly isolate what you're adding.